src/resources/Pipelines/Pipelines.ts

```diff
@@ -16,15 +22,57 @@ export default class Pipelines extends Resource {
         this.associations = new MLAssociations(api);
     }
 
-    listBasicInfo() {
+    getBackendVersion() {

```

Comment:
I'd suggest a less generic name:

```suggestion
    getMLVersion() {
```